### ISSUE TITLE
Should the theme footer show a specific repo's licence?

### ISSUE THREAD

**Issue by ElenaFdR** [publiccodenet/jekyll-theme#55]
The wireframes for the new footer (#51) included a universal licence (CC0) that applied to all content that was shared between our websites (the header and footer). This is set in theme-footer.liquid.

Footer.liquid (not the same file) includes code to dynamically pull a specific repo's license. You can see this on [about.publiccode.net](https://about.publiccode.net/) (under the 'Copyright and license' subheading near the bottom of the page).

As originally designed, #51 led to a webpage appearing to have 2 competing licences without enough contextual clues as to which license was dominant or applied where. To unblock #52, we deleted the universal license from theme-footer.liquid.

This means each webpage should only show 1 license (which seems like the ideal, least confusing choice).

Should the universal footer only show the license for that repo? Will it be a problem if universal content has different licences depending on what repo you're looking at?

**Comment by angelplasma** [comment]
**Below is a potential scheme that follows the logic...**
Cover _all_ content under a single site-wide license... unless it is superseded by a more proximal in-page license.

**The layout scheme:**
1. A site-wide license (e.g. "Creative Commons Zero v1.0 Universal"), visible at the very bottom of the universal page footer.
2a. An **optional** page-specific license, visible at the bottom of the page content (i.e. in the white area just above the footer)
...or if more flexibility is needed...
2b. A visually standardized, in-page license block that can be included wherever appropriate. Multiple such blocks could be included in a single page, if needed.

**Clarifying the hierarchy:**
The hierarchy could be made explicit if necessary, by attaching language to the universal footer license like "All content licensed under Creative Commons Zero v1.0 Universal, unless otherwise noted"

**Potential caveats:**
I'm making a big assumption about a site-wide license being at all applicable/sensical to publiccode.net. If it makes no sense/gives rise to weird conflicts, then perhaps we just need 2a or 2b above.

**Comment by Ainali** [comment]
I like this suggestion and especially the phrase "All content licensed under Creative Commons Zero v1.0 Universal, unless otherwise noted" which would make it easy for us to add images that have other licenses or are non-free even.

**Comment by ElenaFdR** [comment]
I've just discussed this with @bvhme, who's reminded me that one of our founding principles is "We publish under licenses approved by the Open Source Initiative or Creative Commons licenses that do not limit commercial reuse." 
Given this, having a wide open site-wide license with @angelplasma's option 2 would help us defaul to meeting our founding principle (and maybe not angst about this as much in the future).

**Comment by angelplasma** [comment]
Great! @ElenaFdR - Shall I add the license (plus the aforementioned clarification) back into the new footer?

For the in-page solution, people can continue including context-specific licenses in a freeform way, or in the content footer (e.g. as seen on https://standard.publiccode.net)—which we will want to keep in mind as we resolve https://github.com/publiccodenet/publiccode.net/issues/114

**Comment by ElenaFdR** [comment]
@angelplasma, yes to putting this back into the universal footer on theme-footer.liquid.

To summarize our conversation, you propose leaving footer.liquid as an optional block that appears automagically when there's specific license or copyright info, but which no longer includes 'this site on Github' anywhere (as I've been unable to find any users who find this useful).

This means that only very exceptional content (eg a photo with a different licence) would require a note in free text.
